images/bcftools/Dockerfile

```diff
@@ -1,14 +1,26 @@
 FROM debian:buster-slim
 
-ENV MAMBA_ROOT_PREFIX /root/micromamba
-ENV PATH $MAMBA_ROOT_PREFIX/bin:$PATH
-ARG VERSION=${VERSION:-1.16}
+ARG VERSION=${VERSION:-1.20}
 
-RUN apt-get update && apt-get install -y git wget bash bzip2 zip && \
+
+RUN apt-get update && apt-get install -y \
+        bash \
+        bzip2 \
+        gcc \
+        git \
+        libbz2-dev \
+        liblzma-dev \
+        libcurl4-openssl-dev \
+        make \
+        wget \
+        zip \
+        zlib1g-dev && \
     rm -r /var/lib/apt/lists/* && \
     rm -r /var/cache/apt/* && \
-    wget -qO- https://api.anaconda.org/download/conda-forge/micromamba/0.8.2/linux-64/micromamba-0.8.2-he9b6cbd_0.tar.bz2 | tar -xvj -C /usr/local bin/micromamba && \
-    mkdir ${MAMBA_ROOT_PREFIX} && \
-    micromamba install -y --prefix ${MAMBA_ROOT_PREFIX} -c bioconda -c conda-forge \
-    bcftools=${VERSION} && \
-    rm -r /root/micromamba/pkgs
+    git clone --recurse-submodules https://github.com/samtools/htslib.git --branch ${VERSION} --depth 1 && \
+    git clone https://github.com/samtools/bcftools.git --branch ${VERSION} --depth 1 && \
+    cd bcftools && \
+    make && \
+    mv bcftools /usr/local/bin && \
```

Comment:
Use `make install` rather than manually copying a small subset — in particular, you're missing bcftools's plugins.